[windows] Python for windows(store)

Paxxi opened the pull request:
rebuilt most libs because things were outdated and needed a refresh
Some are updated to latest version
adds pillow, python 3.7.5 and pycryptodome

## Description
<!--- Provide a general summary of your change in the Pull Request title above -->
<!--- Describe your change in detail here -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your change -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc -->

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
<!--- Go over all the following points, and put an `X` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ x I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

Paxxi committed:
Rebuild most dependencies

Some are updated to latest version
adds pillow, python 3.7.5 and pycryptodome

Rechi commented on project/BuildDependencies/scripts/0_package.native-win32.list:
```diff
@@ -7,7 +7,7 @@
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 doxygen-1.8.2-win32.7z
-flatc-1.9.0-Win32-v141.7z
+flatc-1.9.0-win10-win32-v142-20191216.7z
```
WindowsStore executable is runabale?

Paxxi replied:
Good catch, copy paste fail

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
-crossguid-fef89a4-win10-Win32-v140.7z
-curl-7.59.0-win10-Win32-v141.7z
+crossguid-fef89a4-win10-win32-v142-20191216.7z
+curl-7.61.1-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
-libxml2-2.9.4-win10-Win32-v141.7z
-libxslt-1.1.29-win10-Win32-v140.7z
-lzo2-2.09-win10-Win32-v140.7z
+fmt-3.0.1-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
-libxslt-1.1.29-win10-Win32-v140.7z
-lzo2-2.09-win10-Win32-v140.7z
+fmt-3.0.1-win10-win32-v142-20191216.7z
+freetype-2.8-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+fstrcmp-0.7-win10-win32-v142-20191216.7z
+lcms2-2.8-win10-win32-v142-20191216.7z
+libaacs-0.9.0-win10-win32-v142-20191216.7z
+libass-0.13.6-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+lcms2-2.8-win10-win32-v142-20191216.7z
+libaacs-0.9.0-win10-win32-v142-20191216.7z
+libass-0.13.6-win10-win32-v142-20191216.7z
+libbluray-1.0.1-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+libaacs-0.9.0-win10-win32-v142-20191216.7z
+libass-0.13.6-win10-win32-v142-20191216.7z
+libbluray-1.0.1-win10-win32-v142-20191216.7z
+libcdio-0.94-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+libass-0.13.6-win10-win32-v142-20191216.7z
+libbluray-1.0.1-win10-win32-v142-20191216.7z
+libcdio-0.94-win10-win32-v142-20191216.7z
+libfribidi-0.19.7-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+libfribidi-0.19.7-win10-win32-v142-20191216.7z
+libiconv-1.16-win10-win32-v142-20191216.7z
+libmicrohttpd-0.9.68-win10-win32-v142-20191216.7z
+libnfs-3.0.0-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
-openssl-1.1.1b-win10-Win32-v141.7z
-pcre-8.40-win10-Win32-v141.7z
-python3-3.7.2-win10-Win32-v141.7z
+mariadb-connector-c-3.0.6-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
-pcre-8.40-win10-Win32-v141.7z
-python3-3.7.2-win10-Win32-v141.7z
+mariadb-connector-c-3.0.6-win10-win32-v142-20191216.7z
+openssl-1.1.1-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
-python3-3.7.2-win10-Win32-v141.7z
+mariadb-connector-c-3.0.6-win10-win32-v142-20191216.7z
+openssl-1.1.1-win10-win32-v142-20191216.7z
+pcre-8.40-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+pycryptodome-3.9.4-win10-win32-v142-20191216.7z
+python-3.7.5-win10-win32-v142-20191216.7z
 rapidjson-1.1.0-win32.7z
-shairplay-ce80e00-win10-Win32-v140.7z
```
Why remove shairplay?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
-taglib-1.11.1-win10-Win32-v140.7z
-tinyxml-2.6.2-win10-Win32-v140.7z
-zlib-1.2.11-win10-Win32-v140.7z
+sqlite-3170000-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
-crossguid-fef89a4-win10-x64-v140.7z
-curl-7.59.0-win10-x64-v141.7z
+crossguid-fef89a4-win10-x64-v142-20191216.7z
+curl-7.61.1-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
-libxml2-2.9.4-win10-x64-v141.7z
-libxslt-1.1.29-win10-x64-v140.7z
-lzo2-2.09-win10-x64-v140.7z
+fmt-3.0.1-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
-libxslt-1.1.29-win10-x64-v140.7z
-lzo2-2.09-win10-x64-v140.7z
+fmt-3.0.1-win10-x64-v142-20191216.7z
+freetype-2.8-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+fmt-3.0.1-win10-win32-v142-20191216.7z
+freetype-2.8-win10-win32-v142-20191216.7z
+fstrcmp-0.7-win10-win32-v142-20191216.7z
+lcms2-2.8-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,33 @@
+libcdio-0.94-win10-win32-v142-20191216.7z
+libfribidi-0.19.7-win10-win32-v142-20191216.7z
+libiconv-1.16-win10-win32-v142-20191216.7z
+libmicrohttpd-0.9.68-win10-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+fmt-3.0.1-win10-x64-v142-20191216.7z
+freetype-2.8-win10-x64-v142-20191216.7z
+fstrcmp-0.7-win10-x64-v142-20191216.7z
+lcms2-2.8-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+fstrcmp-0.7-win10-x64-v142-20191216.7z
+lcms2-2.8-win10-x64-v142-20191216.7z
+libaacs-0.9.0-win10-x64-v142-20191216.7z
+libass-0.13.6-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+lcms2-2.8-win10-x64-v142-20191216.7z
+libaacs-0.9.0-win10-x64-v142-20191216.7z
+libass-0.13.6-win10-x64-v142-20191216.7z
+libbluray-1.0.1-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+libaacs-0.9.0-win10-x64-v142-20191216.7z
+libass-0.13.6-win10-x64-v142-20191216.7z
+libbluray-1.0.1-win10-x64-v142-20191216.7z
+libcdio-0.94-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+libass-0.13.6-win10-x64-v142-20191216.7z
+libbluray-1.0.1-win10-x64-v142-20191216.7z
+libcdio-0.94-win10-x64-v142-20191216.7z
+libfribidi-0.19.7-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+libcdio-0.94-win10-x64-v142-20191216.7z
+libfribidi-0.19.7-win10-x64-v142-20191216.7z
+libiconv-1.16-win10-x64-v142-20191216.7z
+libmicrohttpd-0.9.68-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+libfribidi-0.19.7-win10-x64-v142-20191216.7z
+libiconv-1.16-win10-x64-v142-20191216.7z
+libmicrohttpd-0.9.68-win10-x64-v142-20191216.7z
+libnfs-3.0.0-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
-openssl-1.1.1b-win10-x64-v141.7z
-pcre-8.40-win10-x64-v141.7z
-python3-3.7.2-win10-x64-v141.7z
+mariadb-connector-c-3.0.6-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
-pcre-8.40-win10-x64-v141.7z
-python3-3.7.2-win10-x64-v141.7z
+mariadb-connector-c-3.0.6-win10-x64-v142-20191216.7z
+openssl-1.1.1-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
-python3-3.7.2-win10-x64-v141.7z
+mariadb-connector-c-3.0.6-win10-x64-v142-20191216.7z
+openssl-1.1.1-win10-x64-v142-20191216.7z
+pcre-8.40-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
+pycryptodome-3.9.4-win10-x64-v142-20191216.7z
+python-3.7.5-win10-x64-v142-20191216.7z
 rapidjson-1.1.0-win32.7z
-shairplay-ce80e00-win10-x64-v140.7z
```
Why remove shairplay?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,33 @@
-taglib-1.11.1-win10-x64-v140.7z
-tinyxml-2.6.2-win10-x64-v140.7z
-zlib-1.2.11-win10-x64-v140.7z
+sqlite-3170000-win10-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-dnssd-765.50.9-win32-vc140-v2.7z
-easyhook-2.7.5870.0-win32-vc140-v2.7z
+crossguid-fef89a4-win32-v142-20191216.7z
+curl-7.61.1-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-easyhook-2.7.5870.0-win32-vc140-v2.7z
+crossguid-fef89a4-win32-v142-20191216.7z
+curl-7.61.1-win32-v142-20191216.7z
+dnssd-765.50.9-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-fmt-3.0.1-win32-vc140.7z
-freetype-db5a22-win32-vc140.7z
-fstrcmp-0.7-Win32-v141.7z
+fmt-3.0.1-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-freetype-db5a22-win32-vc140.7z
-fstrcmp-0.7-Win32-v141.7z
+fmt-3.0.1-win32-v142-20191216.7z
+freetype-2.8-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-libmicrohttpd-0.9.55-win32-vc140.7z
-libnfs-3.0.0-Win32-v141.7z
-libplist-1.13.0-win32-vc140.7z
+lcms2-2.8-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-libplist-1.13.0-win32-vc140.7z
+lcms2-2.8-win32-v142-20191216.7z
+libaacs-0.9.0-win32-v142-20191216.7z
+libass-0.13.6-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+lcms2-2.8-win32-v142-20191216.7z
+libaacs-0.9.0-win32-v142-20191216.7z
+libass-0.13.6-win32-v142-20191216.7z
+libbluray-1.0.1-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+libaacs-0.9.0-win32-v142-20191216.7z
+libass-0.13.6-win32-v142-20191216.7z
+libbluray-1.0.1-win32-v142-20191216.7z
+libcdio-0.94-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+libass-0.13.6-win32-v142-20191216.7z
+libbluray-1.0.1-win32-v142-20191216.7z
+libcdio-0.94-win32-v142-20191216.7z
+libcec-4.0.2-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+libbluray-1.0.1-win32-v142-20191216.7z
+libcdio-0.94-win32-v142-20191216.7z
+libcec-4.0.2-win32-v142-20191216.7z
+libfribidi-0.19.7-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+libcec-4.0.2-win32-v142-20191216.7z
+libfribidi-0.19.7-win32-v142-20191216.7z
+libiconv-1.16-win32-v142-20191216.7z
+libjpeg-turbo-1.5.2-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+libfribidi-0.19.7-win32-v142-20191216.7z
+libiconv-1.16-win32-v142-20191216.7z
+libjpeg-turbo-1.5.2-win32-v142-20191216.7z
+libmicrohttpd-0.9.68-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+libiconv-1.16-win32-v142-20191216.7z
+libjpeg-turbo-1.5.2-win32-v142-20191216.7z
+libmicrohttpd-0.9.68-win32-v142-20191216.7z
+libnfs-3.0.0-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
+libjpeg-turbo-1.5.2-win32-v142-20191216.7z
+libmicrohttpd-0.9.68-win32-v142-20191216.7z
+libnfs-3.0.0-win32-v142-20191216.7z
+libplist-1.12-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-openssl-1.1.1b-win10-Win32-v141.7z
-pcre-8.37-win32-vc140-v3.7z
-python3-3.7.1-Win32-v141.7z
+mariadb-connector-c-3.0.6-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-pcre-8.37-win32-vc140-v3.7z
-python3-3.7.1-Win32-v141.7z
+mariadb-connector-c-3.0.6-win32-v142-20191216.7z
+openssl-1.1.1-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-python3-3.7.1-Win32-v141.7z
+mariadb-connector-c-3.0.6-win32-v142-20191216.7z
+openssl-1.1.1-win32-v142-20191216.7z
+pcre-8.40-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,41 @@
-tinyxmlstl-2.6.2-win32-vc140-v2.7z
-zlib-1.2.8-win32-vc140-v3.7z
+shairplay-ce80e00-win32-v142-20191216.7z
+sqlite-3170000-win32-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-dnssd-765.50.9-x64-vc140-v2.7z
-easyhook-2.7.6035.0-x64-vc140.7z
+crossguid-fef89a4-x64-v142-20191216.7z
+curl-7.61.1-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-easyhook-2.7.6035.0-x64-vc140.7z
+crossguid-fef89a4-x64-v142-20191216.7z
+curl-7.61.1-x64-v142-20191216.7z
+dnssd-765.50.9-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-libxml2-2.9.4-x64-vc140.7z
-libxslt-1.1.29-x64-vc140.7z
-lzo-2.09-x64-vc140.7z
+fmt-3.0.1-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-libxslt-1.1.29-x64-vc140.7z
-lzo-2.09-x64-vc140.7z
+fmt-3.0.1-x64-v142-20191216.7z
+freetype-2.8-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+fmt-3.0.1-x64-v142-20191216.7z
+freetype-2.8-x64-v142-20191216.7z
+fstrcmp-0.7-x64-v142-20191216.7z
+lcms2-2.8-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+fstrcmp-0.7-x64-v142-20191216.7z
+lcms2-2.8-x64-v142-20191216.7z
+libaacs-0.9.0-x64-v142-20191216.7z
+libass-0.13.6-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+lcms2-2.8-x64-v142-20191216.7z
+libaacs-0.9.0-x64-v142-20191216.7z
+libass-0.13.6-x64-v142-20191216.7z
+libbluray-1.0.1-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+libaacs-0.9.0-x64-v142-20191216.7z
+libass-0.13.6-x64-v142-20191216.7z
+libbluray-1.0.1-x64-v142-20191216.7z
+libcdio-0.94-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+libass-0.13.6-x64-v142-20191216.7z
+libbluray-1.0.1-x64-v142-20191216.7z
+libcdio-0.94-x64-v142-20191216.7z
+libcec-4.0.2-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+libbluray-1.0.1-x64-v142-20191216.7z
+libcdio-0.94-x64-v142-20191216.7z
+libcec-4.0.2-x64-v142-20191216.7z
+libfribidi-0.19.7-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+libcec-4.0.2-x64-v142-20191216.7z
+libfribidi-0.19.7-x64-v142-20191216.7z
+libiconv-1.16-x64-v142-20191216.7z
+libmicrohttpd-0.9.68-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+libfribidi-0.19.7-x64-v142-20191216.7z
+libiconv-1.16-x64-v142-20191216.7z
+libmicrohttpd-0.9.68-x64-v142-20191216.7z
+libnfs-3.0.0-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
+libiconv-1.16-x64-v142-20191216.7z
+libmicrohttpd-0.9.68-x64-v142-20191216.7z
+libnfs-3.0.0-x64-v142-20191216.7z
+libplist-1.12-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-openssl-1.1.1-x64-v141.7z
-pcre-8.40-x64-vc140.7z
-python3-3.7.1-x64-v141.7z
+mariadb-connector-c-3.0.6-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-pcre-8.40-x64-vc140.7z
-python3-3.7.1-x64-v141.7z
+mariadb-connector-c-3.0.6-x64-v142-20191216.7z
+openssl-1.1.1-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-python3-3.7.1-x64-v141.7z
+mariadb-connector-c-3.0.6-x64-v142-20191216.7z
+openssl-1.1.1-x64-v142-20191216.7z
+pcre-8.40-x64-v142-20191216.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,38 @@
-tinyxmlstl-2.6.2-x64-vc140.7z
-zlib-1.2.11-x64-v141.7z
+shairplay-ce80e00-x64-v142-20191216.7z
+sqlite-3170000-x64-v142-20191216.7z
```
newer version available

Rechi reviewed (changes requested):
- updating dependencies for WindowsStore arm is missing
- many versions of the depdendencies are outdated
- Is v142 compatible with v140 and v141?
- Are the rebuild dependencies still compatible with Windows 7?

Paxxi commented:
I avoided bumping versions of most libraries to minimize the risk. This was only supposed to be a python fix but it spiraled out of control a bit 😀

Plan is to bump versions next but as a separate pr

Paxxi committed:
copy paste mistake

Paxxi committed:
Fix bad packaging

Remove the iterator_debug_level setting and default to 2 as on
all other windows platforms

bump libbluray to 1.1.2
bump lzo to 2.10
include libbdplus for libbluray

Paxxi commented:
jenkins build this please

Rechi commented on project/BuildDependencies/scripts/0_package.native-win32.list:
```diff
@@ -7,7 +7,7 @@
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 doxygen-1.8.2-win32.7z
-flatc-1.9.0-Win32-v141.7z
+flatc-1.9.0-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-crossguid-fef89a4-win10-Win32-v140.7z
-curl-7.59.0-win10-Win32-v141.7z
+crossguid-fef89a4-win10-win32-v141-20191217.7z
+curl-7.61.1-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-libxml2-2.9.4-win10-Win32-v141.7z
-libxslt-1.1.29-win10-Win32-v140.7z
-lzo2-2.09-win10-Win32-v140.7z
+fmt-3.0.1-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-libxslt-1.1.29-win10-Win32-v140.7z
-lzo2-2.09-win10-Win32-v140.7z
+fmt-3.0.1-win10-win32-v141-20191217.7z
+freetype-2.8-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+fmt-3.0.1-win10-win32-v141-20191217.7z
+freetype-2.8-win10-win32-v141-20191217.7z
+fstrcmp-0.7-win10-win32-v141-20191217.7z
+lcms2-2.8-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+fstrcmp-0.7-win10-win32-v141-20191217.7z
+lcms2-2.8-win10-win32-v141-20191217.7z
+libaacs-0.9.0-win10-win32-v141-20191217.7z
+libass-0.13.6-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-libiconv-1.15-win10-Win32-v141.7z
-libmicrohttpd-0.9.55-win10-Win32-v140.7z
-libnfs-3.0.0-win10-Win32-v141.7z
-libplist-1.12-win10-Win32-v140.7z
```
Why remove libplist?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+lcms2-2.8-win10-win32-v141-20191217.7z
+libaacs-0.9.0-win10-win32-v141-20191217.7z
+libass-0.13.6-win10-win32-v141-20191217.7z
+libbdplus-0.1.2-win10-win32-v141-20191217.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+libass-0.13.6-win10-win32-v141-20191217.7z
+libbdplus-0.1.2-win10-win32-v141-20191217.7z
+libbluray-1.1.2-win10-win32-v141-20191217.7z
+libcdio-0.94-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+libbdplus-0.1.2-win10-win32-v141-20191217.7z
+libbluray-1.1.2-win10-win32-v141-20191217.7z
+libcdio-0.94-win10-win32-v141-20191217.7z
+libfribidi-0.19.7-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+libcdio-0.94-win10-win32-v141-20191217.7z
+libfribidi-0.19.7-win10-win32-v141-20191217.7z
+libiconv-1.16-win10-win32-v141-20191217.7z
+libmicrohttpd-0.9.68-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+libfribidi-0.19.7-win10-win32-v141-20191217.7z
+libiconv-1.16-win10-win32-v141-20191217.7z
+libmicrohttpd-0.9.68-win10-win32-v141-20191217.7z
+libnfs-3.0.0-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-openssl-1.1.1b-win10-Win32-v141.7z
-pcre-8.40-win10-Win32-v141.7z
-python3-3.7.2-win10-Win32-v141.7z
+mariadb-connector-c-3.0.6-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-pcre-8.40-win10-Win32-v141.7z
-python3-3.7.2-win10-Win32-v141.7z
+mariadb-connector-c-3.0.6-win10-win32-v141-20191217.7z
+openssl-1.1.1-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-python3-3.7.2-win10-Win32-v141.7z
+mariadb-connector-c-3.0.6-win10-win32-v141-20191217.7z
+openssl-1.1.1-win10-win32-v141-20191217.7z
+pcre-8.40-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-taglib-1.11.1-win10-Win32-v140.7z
-tinyxml-2.6.2-win10-Win32-v140.7z
-zlib-1.2.11-win10-Win32-v140.7z
+sqlite-3170000-win10-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-crossguid-fef89a4-win10-x64-v140.7z
-curl-7.59.0-win10-x64-v141.7z
+crossguid-fef89a4-win10-x64-v141-20191217.7z
+curl-7.61.1-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-libxml2-2.9.4-win10-x64-v141.7z
-libxslt-1.1.29-win10-x64-v140.7z
-lzo2-2.09-win10-x64-v140.7z
+fmt-3.0.1-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-libxslt-1.1.29-win10-x64-v140.7z
-lzo2-2.09-win10-x64-v140.7z
+fmt-3.0.1-win10-x64-v141-20191217.7z
+freetype-2.8-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+fmt-3.0.1-win10-x64-v141-20191217.7z
+freetype-2.8-win10-x64-v141-20191217.7z
+fstrcmp-0.7-win10-x64-v141-20191217.7z
+lcms2-2.8-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+fstrcmp-0.7-win10-x64-v141-20191217.7z
+lcms2-2.8-win10-x64-v141-20191217.7z
+libaacs-0.9.0-win10-x64-v141-20191217.7z
+libass-0.13.6-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-libiconv-1.15-win10-x64-v141.7z
-libmicrohttpd-0.9.55-win10-x64-v140.7z
-libnfs-3.0.0-win10-x64-v141.7z
-libplist-1.12-win10-x64-v140.7z
```
Why remove libplist?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+freetype-2.8-win10-x64-v141-20191217.7z
+fstrcmp-0.7-win10-x64-v141-20191217.7z
+lcms2-2.8-win10-x64-v141-20191217.7z
+libaacs-0.9.0-win10-x64-v141-20191217.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+libass-0.13.6-win10-x64-v141-20191217.7z
+libbdplus-0.1.2-win10-x64-v141-20191217.7z
+libbluray-1.1.2-win10-x64-v141-20191217.7z
+libcdio-0.94-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+libbdplus-0.1.2-win10-x64-v141-20191217.7z
+libbluray-1.1.2-win10-x64-v141-20191217.7z
+libcdio-0.94-win10-x64-v141-20191217.7z
+libfribidi-0.19.7-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+libcdio-0.94-win10-x64-v141-20191217.7z
+libfribidi-0.19.7-win10-x64-v141-20191217.7z
+libiconv-1.16-win10-x64-v141-20191217.7z
+libmicrohttpd-0.9.68-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+libfribidi-0.19.7-win10-x64-v141-20191217.7z
+libiconv-1.16-win10-x64-v141-20191217.7z
+libmicrohttpd-0.9.68-win10-x64-v141-20191217.7z
+libnfs-3.0.0-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-openssl-1.1.1b-win10-x64-v141.7z
-pcre-8.40-win10-x64-v141.7z
-python3-3.7.2-win10-x64-v141.7z
+mariadb-connector-c-3.0.6-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-pcre-8.40-win10-x64-v141.7z
-python3-3.7.2-win10-x64-v141.7z
+mariadb-connector-c-3.0.6-win10-x64-v141-20191217.7z
+openssl-1.1.1-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-taglib-1.11.1-win10-x64-v140.7z
-tinyxml-2.6.2-win10-x64-v140.7z
-zlib-1.2.11-win10-x64-v140.7z
+sqlite-3170000-win10-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-dnssd-765.50.9-win32-vc140-v2.7z
-easyhook-2.7.5870.0-win32-vc140-v2.7z
+crossguid-fef89a4-win32-v141-20191217.7z
+curl-7.61.1-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-easyhook-2.7.5870.0-win32-vc140-v2.7z
+crossguid-fef89a4-win32-v141-20191217.7z
+curl-7.61.1-win32-v141-20191217.7z
+dnssd-765.50.9-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-fmt-3.0.1-win32-vc140.7z
-freetype-db5a22-win32-vc140.7z
-fstrcmp-0.7-Win32-v141.7z
+fmt-3.0.1-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-freetype-db5a22-win32-vc140.7z
-fstrcmp-0.7-Win32-v141.7z
+fmt-3.0.1-win32-v141-20191217.7z
+freetype-2.8-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-libmicrohttpd-0.9.55-win32-vc140.7z
-libnfs-3.0.0-Win32-v141.7z
-libplist-1.13.0-win32-vc140.7z
+lcms2-2.8-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-libplist-1.13.0-win32-vc140.7z
+lcms2-2.8-win32-v141-20191217.7z
+libaacs-0.9.0-win32-v141-20191217.7z
+libass-0.13.6-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+lcms2-2.8-win32-v141-20191217.7z
+libaacs-0.9.0-win32-v141-20191217.7z
+libass-0.13.6-win32-v141-20191217.7z
+libbdplus-0.1.2-win32-v141-20191217.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libass-0.13.6-win32-v141-20191217.7z
+libbdplus-0.1.2-win32-v141-20191217.7z
+libbluray-1.1.2-win32-v141-20191217.7z
+libcdio-0.94-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libbdplus-0.1.2-win32-v141-20191217.7z
+libbluray-1.1.2-win32-v141-20191217.7z
+libcdio-0.94-win32-v141-20191217.7z
+libcec-4.0.2-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libbluray-1.1.2-win32-v141-20191217.7z
+libcdio-0.94-win32-v141-20191217.7z
+libcec-4.0.2-win32-v141-20191217.7z
+libfribidi-0.19.7-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libcec-4.0.2-win32-v141-20191217.7z
+libfribidi-0.19.7-win32-v141-20191217.7z
+libiconv-1.16-win32-v141-20191217.7z
+libjpeg-turbo-1.5.2-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libfribidi-0.19.7-win32-v141-20191217.7z
+libiconv-1.16-win32-v141-20191217.7z
+libjpeg-turbo-1.5.2-win32-v141-20191217.7z
+libmicrohttpd-0.9.68-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libiconv-1.16-win32-v141-20191217.7z
+libjpeg-turbo-1.5.2-win32-v141-20191217.7z
+libmicrohttpd-0.9.68-win32-v141-20191217.7z
+libnfs-3.0.0-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libjpeg-turbo-1.5.2-win32-v141-20191217.7z
+libmicrohttpd-0.9.68-win32-v141-20191217.7z
+libnfs-3.0.0-win32-v141-20191217.7z
+libplist-1.12-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-openssl-1.1.1b-win10-Win32-v141.7z
-pcre-8.37-win32-vc140-v3.7z
-python3-3.7.1-Win32-v141.7z
+mariadb-connector-c-3.0.6-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-pcre-8.37-win32-vc140-v3.7z
-python3-3.7.1-Win32-v141.7z
+mariadb-connector-c-3.0.6-win32-v141-20191217.7z
+openssl-1.1.1-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-python3-3.7.1-Win32-v141.7z
+mariadb-connector-c-3.0.6-win32-v141-20191217.7z
+openssl-1.1.1-win32-v141-20191217.7z
+pcre-8.40-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-tinyxmlstl-2.6.2-win32-vc140-v2.7z
-zlib-1.2.8-win32-vc140-v3.7z
+shairplay-ce80e00-win32-v141-20191217.7z
+sqlite-3170000-win32-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-dnssd-765.50.9-x64-vc140-v2.7z
-easyhook-2.7.6035.0-x64-vc140.7z
+crossguid-fef89a4-x64-v141-20191217.7z
+curl-7.61.1-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-easyhook-2.7.6035.0-x64-vc140.7z
+crossguid-fef89a4-x64-v141-20191217.7z
+curl-7.61.1-x64-v141-20191217.7z
+dnssd-765.50.9-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-libxml2-2.9.4-x64-vc140.7z
-libxslt-1.1.29-x64-vc140.7z
-lzo-2.09-x64-vc140.7z
+fmt-3.0.1-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-libxslt-1.1.29-x64-vc140.7z
-lzo-2.09-x64-vc140.7z
+fmt-3.0.1-x64-v141-20191217.7z
+freetype-2.8-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+fmt-3.0.1-x64-v141-20191217.7z
+freetype-2.8-x64-v141-20191217.7z
+fstrcmp-0.7-x64-v141-20191217.7z
+lcms2-2.8-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+fstrcmp-0.7-x64-v141-20191217.7z
+lcms2-2.8-x64-v141-20191217.7z
+libaacs-0.9.0-x64-v141-20191217.7z
+libass-0.13.6-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+lcms2-2.8-x64-v141-20191217.7z
+libaacs-0.9.0-x64-v141-20191217.7z
+libass-0.13.6-x64-v141-20191217.7z
+libbdplus-0.1.2-x64-v141-20191217.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libass-0.13.6-x64-v141-20191217.7z
+libbdplus-0.1.2-x64-v141-20191217.7z
+libbluray-1.1.2-x64-v141-20191217.7z
+libcdio-0.94-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libbdplus-0.1.2-x64-v141-20191217.7z
+libbluray-1.1.2-x64-v141-20191217.7z
+libcdio-0.94-x64-v141-20191217.7z
+libcec-4.0.2-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libbluray-1.1.2-x64-v141-20191217.7z
+libcdio-0.94-x64-v141-20191217.7z
+libcec-4.0.2-x64-v141-20191217.7z
+libfribidi-0.19.7-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libcec-4.0.2-x64-v141-20191217.7z
+libfribidi-0.19.7-x64-v141-20191217.7z
+libiconv-1.16-x64-v141-20191217.7z
+libmicrohttpd-0.9.68-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libfribidi-0.19.7-x64-v141-20191217.7z
+libiconv-1.16-x64-v141-20191217.7z
+libmicrohttpd-0.9.68-x64-v141-20191217.7z
+libnfs-3.0.0-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libiconv-1.16-x64-v141-20191217.7z
+libmicrohttpd-0.9.68-x64-v141-20191217.7z
+libnfs-3.0.0-x64-v141-20191217.7z
+libplist-1.12-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-openssl-1.1.1-x64-v141.7z
-pcre-8.40-x64-vc140.7z
-python3-3.7.1-x64-v141.7z
+mariadb-connector-c-3.0.6-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-pcre-8.40-x64-vc140.7z
-python3-3.7.1-x64-v141.7z
+mariadb-connector-c-3.0.6-x64-v141-20191217.7z
+openssl-1.1.1-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-python3-3.7.1-x64-v141.7z
+mariadb-connector-c-3.0.6-x64-v141-20191217.7z
+openssl-1.1.1-x64-v141-20191217.7z
+pcre-8.40-x64-v141-20191217.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-tinyxmlstl-2.6.2-x64-vc140.7z
-zlib-1.2.11-x64-v141.7z
+shairplay-ce80e00-x64-v141-20191217.7z
+sqlite-3170000-x64-v141-20191217.7z
```
newer version available

Rechi commented on xbmc/guilib/TextureBundleXBT.cpp:
```diff
@@ -26,10 +26,8 @@
-#pragma comment(lib, "lzo2d.lib")
 #else
-#pragma comment(lib, "lzo2-no_idb.lib")
+#pragma comment(lib, "lzo2d.lib")
```
This has to be handled by the build system.

Paxxi replied:
yeah that would be better, that's for another PR though

Rechi replied:
no, remove the hack in this PR

Rechi reviewed (changes requested):
- updating dependencies for WindowsStore arm is missing
- many versions of the dependencies are outdated
- Are the rebuild dependencies still compatible with Windows 7?

Paxxi commented:
> * updating dependencies for WindowsStore arm is missing

Yes, arm requires some more effort and not all libs build for it yet. Afaik arm was only a supported target because @afedchin wanted to run it on his phone? :)

> * many versions of the dependencies are outdated
> * Are the rebuild dependencies still compatible with Windows 7?
They should be, I've kept _WIN32_WINNT, NTDDI_VERSION etc at version 601 for desktop builds and set it to our minimum sdk version for store builds. I don't have a win7 machine so it is untested and if things fail I'll have to fix that

My plan is roughly
1. get this merged to unblock x64-uwp
2. bump everything possible to latest version(maybe not bump python to 3.8 though), add a couple of libs such as brotli and nghttp2 for curl to enable support for modern stuff
3. look at arm-uwp, arm64 and arm64-uwp

Getting this far has taken a couple of months and my spare time will be a lot more limited after the holidays. It feels reasonable to do the major platforms first and if I run out of time things at least work or better yet someone can continue the work and finish up the remaining stuff.

afedchin commented:
@Paxxi just in case if you missed something https://github.com/afedchin/kodi-deps/tree/uwp

Rechi commented:
Either fix arm WindowsStore at the same time as the other WindowsStore architectures are fixed or drop the support.

Where have you kept `_WIN32_WINNT` and `NTDDI_VERSION` defines?

> [...] my spare time will be a lot more limited after the holidays.

So you might not be available to rebuild dependencies again and updating versions won't happen soon?

Paxxi committed:
Bump libraries

fix some bugs finding and using the newer versions

Paxxi commented:
Ok I bumped all libraries and fixed the arm errors so those are updated as well.

All the work can be found here https://github.com/Paxxi/kodi-deps/tree/work . Patches are mainly in the different repos linked. There's a script `DoRelease.ps1` that takes care of building everything in the correct configurations and packaging it up(Thanks for adding that feature 😄 ). My work branch should be merged into xbmc/kodi-deps but it's quite ugly right now as I've just commited things here or there.

Paxxi commented:
jenkins build this please

Rechi commented on cmake/platform/windows/defines.txt:
```diff
@@ -1 +1 @@
--DTARGET_WINDOWS -DTARGET_WINDOWS_DESKTOP -DNOMINMAX -D_CRT_SECURE_NO_WARNINGS -D_WINSOCKAPI_
\ No newline at end of file
+-DTARGET_WINDOWS -DTARGET_WINDOWS_DESKTOP -DNOMINMAX -D_CRT_SECURE_NO_WARNINGS -D_WINSOCKAPI_ -DFRIBIDI_STATIC
```
this is the wrong place to add a define for a library

Rechi commented on cmake/platform/windowsstore/defines.txt:
```diff
@@ -1 +1 @@
--DTARGET_WINDOWS -DTARGET_WINDOWS_STORE -DNOMINMAX -D_CRT_SECURE_NO_WARNINGS -D_WINSOCKAPI_
\ No newline at end of file
+-DTARGET_WINDOWS -DTARGET_WINDOWS_STORE -DNOMINMAX -D_CRT_SECURE_NO_WARNINGS -D_WINSOCKAPI_ -DFRIBIDI_STATIC
```
this is the wrong place to add a define for a library

Rechi commented on cmake/scripts/windows/ArchSetup.cmake:
```diff
@@ -45,12 +45,9 @@ set(SYSTEM_DEFINES -DWIN32_LEAN_AND_MEAN -DNOMINMAX -DHAS_DX -D__STDC_CONSTANT_M
                    -DPLT_HTTP_DEFAULT_USER_AGENT="UPnP/1.0 DLNADOC/1.50 Kodi"
                    -DPLT_HTTP_DEFAULT_SERVER="UPnP/1.0 DLNADOC/1.50 Kodi"
                    -DUNICODE -D_UNICODE
+                   -DFRIBIDI_STATIC
```
this is the wrong place to add a define for a library

Paxxi replied:
Why? These lines are for libraries
```
-DTAGLIB_STATIC -DNPT_CONFIG_ENABLE_LOGGING
-DPLT_HTTP_DEFAULT_USER_AGENT="UPnP/1.0 DLNADOC/1.50 Kodi"
-DPLT_HTTP_DEFAULT_SERVER="UPnP/1.0 DLNADOC/1.50 Kodi"
```

Rechi replied:
`TAGLIB_STATIC` is wrong too, because you tell in general that the library is built static on windows, but that's not true. A non static variant would work too, but then the define is wrong.
`PLT_HTTP_DEFAULT_USER_AGENT` and `PLT_HTTP_DEFAULT_SERVER` should be defined at a common place for all platforms, because they are currently only defined for Windows and WindowsStore.

Rechi commented on cmake/scripts/windowsstore/ArchSetup.cmake:
```diff
@@ -71,6 +71,7 @@ set(SYSTEM_DEFINES -DWIN32_LEAN_AND_MEAN -DNOMINMAX -DHAS_DX -D__STDC_CONSTANT_M
                    -DPLT_HTTP_DEFAULT_USER_AGENT="UPnP/1.0 DLNADOC/1.50 Kodi"
                    -DPLT_HTTP_DEFAULT_SERVER="UPnP/1.0 DLNADOC/1.50 Kodi"
                    -DUNICODE -D_UNICODE
+                   -DFRIBIDI_STATIC
```
this is the wrong place to add a define for a library

Rechi commented on project/BuildDependencies/scripts/0_package.native-win32.list:
```diff
@@ -7,7 +7,7 @@
 ; -> ...
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 doxygen-1.8.2-win32.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.native-win32.list:
```diff
@@ -7,7 +7,7 @@
 doxygen-1.8.2-win32.7z
-flatc-1.9.0-Win32-v141.7z
+flatc-1.11.0-win32-v141-20191222.7z
 jsonschemabuilder-1.0.0-win32-3.7z
```
newer version available

Paxxi replied:
it's not part of kodi-deps so there's no reason for me to touch that now

Rechi replied:
It's part of depends and as Windows doesn't build dependencies from source that one has to rebuild and bundled too.

Rechi commented on project/BuildDependencies/scripts/0_package.native-win32.list:
```diff
@@ -7,7 +7,7 @@
-flatc-1.9.0-Win32-v141.7z
+flatc-1.11.0-win32-v141-20191222.7z
 jsonschemabuilder-1.0.0-win32-3.7z
 swig-3.0.10-win32.7z
```
newer version available

Paxxi replied:
it's not part of kodi-deps so there's no reason for me to touch that now

Rechi replied:
It's part of depends and as Windows doesn't build dependencies from source that one has to rebuild and bundled too.

Rechi commented on project/BuildDependencies/scripts/0_package.native-win32.list:
```diff
@@ -7,7 +7,7 @@
+flatc-1.11.0-win32-v141-20191222.7z
 jsonschemabuilder-1.0.0-win32-3.7z
 swig-3.0.10-win32.7z
 texturepacker-1.1.1-win32.7z
```
newer version available

Paxxi replied:
it's not part of kodi-deps so there's no reason for me to touch that now

Rechi replied:
It's part of depends and as Windows doesn't build dependencies from source that one has to rebuild and bundled too.

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-arm.list:
```diff
@@ -1,37 +1,39 @@
-taglib-1.11.1-win10-ARM-v140.7z
-tinyxml-2.6.2-win10-ARM-v140.7z
-zlib-1.2.11-win10-ARM-v140.7z
+crossguid-2.0.3-win10-arm-v141-20191222.7z
```
Where did you get version 2.0.3 from?

Paxxi replied:
I have no clue really. Somehow I got 0.2.2 to be 2.0.3, will fix

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-arm.list:
```diff
@@ -1,37 +1,39 @@
+lcms2-2.9-win10-arm-v141-20191222.7z
+libaacs-0.9.0-win10-arm-v141-20191222.7z
+libass-0.14.0-win10-arm-v141-20191222.7z
+libbdplus-0.1.2-win10-arm-v141-20191222.7z
```
Should this be bundled with Kodi?

Paxxi replied:
I don't see why not, it's used by libbluray so we need it

Rechi replied:
No libbluray works fine without is. Also it only dynamically loads this library so one could place it at the correct location and it will work too, except for WindowsStore I know.
I'm just not sure if it's a good idea to bundle it.

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-arm.list:
```diff
@@ -1,37 +1,39 @@
+libass-0.14.0-win10-arm-v141-20191222.7z
+libbdplus-0.1.2-win10-arm-v141-20191222.7z
+libbluray-1.1.2-win10-arm-v141-20191222.7z
+libcdio-0.94-win10-arm-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-arm.list:
```diff
@@ -1,37 +1,39 @@
+libcdio-0.94-win10-arm-v141-20191222.7z
+libfribidi-1.0.8-win10-arm-v141-20191222.7z
+libiconv-1.16-win10-arm-v141-20191222.7z
+libmicrohttpd-0.9.68-win10-arm-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-arm.list:
```diff
@@ -1,37 +1,39 @@
+libxml2-2.9.9-win10-arm-v141-20191222.7z
+libxslt-1.1.34-win10-arm-v141-20191222.7z
+lzo2-2.10-win10-arm-v141-20191222.7z
+mini_wdk-10.0.14393.0-arm.7z
```
Is this still needed?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-arm.list:
```diff
@@ -1,37 +1,39 @@
+libxslt-1.1.34-win10-arm-v141-20191222.7z
+lzo2-2.10-win10-arm-v141-20191222.7z
+mini_wdk-10.0.14393.0-arm.7z
+mariadb-connector-c-3.0.6-win10-arm-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-libxml2-2.9.4-win10-Win32-v141.7z
-libxslt-1.1.29-win10-Win32-v140.7z
-lzo2-2.09-win10-Win32-v140.7z
+crossguid-2.0.3-win10-win32-v141-20191222.7z
```
Where did you get version 2.0.3 from?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+lcms2-2.9-win10-win32-v141-20191222.7z
+libaacs-0.9.0-win10-win32-v141-20191222.7z
+libass-0.14.0-win10-win32-v141-20191222.7z
+libbdplus-0.1.2-win10-win32-v141-20191222.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+libass-0.14.0-win10-win32-v141-20191222.7z
+libbdplus-0.1.2-win10-win32-v141-20191222.7z
+libbluray-1.1.2-win10-win32-v141-20191222.7z
+libcdio-0.94-win10-win32-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
-taglib-1.11.1-win10-Win32-v140.7z
-tinyxml-2.6.2-win10-Win32-v140.7z
-zlib-1.2.11-win10-Win32-v140.7z
+mariadb-connector-c-3.0.6-win10-win32-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-win32.list:
```diff
@@ -6,33 +6,34 @@
+libcdio-0.94-win10-win32-v141-20191222.7z
+libfribidi-1.0.8-win10-win32-v141-20191222.7z
+libiconv-1.16-win10-win32-v141-20191222.7z
+libmicrohttpd-0.9.68-win10-win32-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-libxml2-2.9.4-win10-x64-v141.7z
-libxslt-1.1.29-win10-x64-v140.7z
-lzo2-2.09-win10-x64-v140.7z
+crossguid-2.0.3-win10-x64-v141-20191222.7z
```
Where did you get version 2.0.3 from?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+lcms2-2.9-win10-x64-v141-20191222.7z
+libaacs-0.9.0-win10-x64-v141-20191222.7z
+libass-0.14.0-win10-x64-v141-20191222.7z
+libbdplus-0.1.2-win10-x64-v141-20191222.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+libass-0.14.0-win10-x64-v141-20191222.7z
+libbdplus-0.1.2-win10-x64-v141-20191222.7z
+libbluray-1.1.2-win10-x64-v141-20191222.7z
+libcdio-0.94-win10-x64-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
+libcdio-0.94-win10-x64-v141-20191222.7z
+libfribidi-1.0.8-win10-x64-v141-20191222.7z
+libiconv-1.16-win10-x64-v141-20191222.7z
+libmicrohttpd-0.9.68-win10-x64-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win10-x64.list:
```diff
@@ -6,33 +6,34 @@
-taglib-1.11.1-win10-x64-v140.7z
-tinyxml-2.6.2-win10-x64-v140.7z
-zlib-1.2.11-win10-x64-v140.7z
+mariadb-connector-c-3.0.6-win10-x64-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-fmt-3.0.1-win32-vc140.7z
-freetype-db5a22-win32-vc140.7z
-fstrcmp-0.7-Win32-v141.7z
+crossguid-2.0.3-win32-v141-20191222.7z
```
Where did you get version 2.0.3 from?

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-libxml2-2.9.4-x64-vc140.7z
-libxslt-1.1.29-x64-vc140.7z
-lzo-2.09-x64-vc140.7z
+crossguid-2.0.3-x64-v141-20191222.7z
```
Where did you get version 2.0.3 from?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+fmt-6.1.2-win32-v141-20191222.7z
+freetype-2.10.1-win32-v141-20191222.7z
+fstrcmp-0.7-win32-v141-20191222.7z
 giflib-5.1.4-win32-vc140.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+lcms2-2.9-win32-v141-20191222.7z
+libaacs-0.9.0-win32-v141-20191222.7z
+libass-0.14.0-win32-v141-20191222.7z
+libbdplus-0.1.2-win32-v141-20191222.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libass-0.14.0-win32-v141-20191222.7z
+libbdplus-0.1.2-win32-v141-20191222.7z
+libbluray-1.1.2-win32-v141-20191222.7z
+libcdio-0.94-win32-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libbdplus-0.1.2-win32-v141-20191222.7z
+libbluray-1.1.2-win32-v141-20191222.7z
+libcdio-0.94-win32-v141-20191222.7z
+libcec-4.0.2-win32-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
+libfribidi-1.0.8-win32-v141-20191222.7z
+libiconv-1.16-win32-v141-20191222.7z
+libjpeg-turbo-2.0.3-win32-v141-20191222.7z
+libmicrohttpd-0.9.68-win32-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-win32.list:
```diff
@@ -6,39 +6,42 @@
-taglib-1.11.1_80df30-win32-vc140.7z
-tinyxmlstl-2.6.2-win32-vc140-v2.7z
-zlib-1.2.8-win32-vc140-v3.7z
+mariadb-connector-c-3.0.6-win32-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+lcms2-2.9-x64-v141-20191222.7z
+libaacs-0.9.0-x64-v141-20191222.7z
+libass-0.14.0-x64-v141-20191222.7z
+libbdplus-0.1.2-x64-v141-20191222.7z
```
Should this be bundled with Kodi?

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libass-0.14.0-x64-v141-20191222.7z
+libbdplus-0.1.2-x64-v141-20191222.7z
+libbluray-1.1.2-x64-v141-20191222.7z
+libcdio-0.94-x64-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libbdplus-0.1.2-x64-v141-20191222.7z
+libbluray-1.1.2-x64-v141-20191222.7z
+libcdio-0.94-x64-v141-20191222.7z
+libcec-4.0.2-x64-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
+libcec-4.0.2-x64-v141-20191222.7z
+libfribidi-1.0.8-x64-v141-20191222.7z
+libiconv-1.16-x64-v141-20191222.7z
+libmicrohttpd-0.9.68-x64-v141-20191222.7z
```
newer version available

Rechi commented on project/BuildDependencies/scripts/0_package.target-x64.list:
```diff
@@ -6,36 +6,39 @@
-taglib-1.11.1-x64-vc140.7z
-tinyxmlstl-2.6.2-x64-vc140.7z
-zlib-1.2.11-x64-v141.7z
+mariadb-connector-c-3.0.6-x64-v141-20191222.7z
```
newer version available

Rechi commented on xbmc/utils/StringUtils.cpp:
```diff
@@ -16,7 +16,11 @@
+#include <guid.hpp>
+#else
 #include <guid.h>
+#endif
```
no platform hacks, if newer versions ship `guid.hpp`

Paxxi replied:
feel free to fix the other platforms, that's not something I intend to do. There's enough work done in this PR

Rechi replied:
Either use a version of crossguid which is compatible with the current code or adjust the code the right way, without hacks.

Rechi reviewed (changes requested):
- some versions of the dependencies are outdated
- Why remove liplist and shairplay for WindosStore?
- Are the rebuild dependencies still compatible with Windows 7?
- no platform hacks

Paxxi commented:
shairplay relies on mddns afaik which we don't have in windows store so it's not really usable even if it builds. libplist is only used by shairplay so it's not needed.

I don't see a reason why they shouldn't work on windows 7 but I haven't tested it so if there's any errors that will have to be fixed when discovered.

afedchin commented:
> shairplay relies on mddns afaik which we don't have in windows store so it's not really usable even if it builds.

iirc because msdns will no work in uwp environment because cross process connections are forbiden.
